srcDLL/DllMain.cpp

```diff
@@ -161,10 +182,15 @@ HINSTANCE __stdcall LoadShell(LPCSTR lpLibFileName)
 
 	if (hInstance) // if good, then setup the language data and call the mod
```

Comment:
I think the comment associated with the if statement on hInstance would maybe best be deleted since we are not actually setting up language data. 

It could be moved as a comment on the function OnLoadShell since the work is now done there.